Tools/TestWebKitAPI/WebTransportServer.mm

```diff
@@ -29,11 +29,27 @@
 #import "HTTPServer.h"
 #import "Utilities.h"
 #import <pal/spi/cf/CFNetworkSPI.h>
+#import <wtf/BlockPtr.h>
 
 namespace TestWebKitAPI {
 
-WebTransportServer::WebTransportServer()
+struct WebTransportServer::Data : public RefCounted<WebTransportServer::Data> {
+    static Ref<Data> create(Function<Task(Connection)>&& connectionHandler) { return adoptRef(*new Data(WTFMove(connectionHandler))); }
+    Data(Function<Task(Connection)>&& connectionHandler)
+        : connectionHandler(WTFMove(connectionHandler)) { }
+
+    Function<Task(Connection)> connectionHandler;
+    RetainPtr<nw_listener_t> listener;
+    Vector<RetainPtr<nw_connection_group_t>> connectionGroups;
+    Vector<RetainPtr<nw_connection_t>> connections;
+    Vector<CoroutineHandle<Task::promise_type>> coroutineHandles;
+};
+
+WebTransportServer::WebTransportServer(Function<Task(Connection)>&& connectionHandler)
+    : m_data(Data::create(WTFMove(connectionHandler)))
 {
+    Ref data { m_data };
```

Comment:
`protectedData`?

Reply:
This is only needed for the block captures.  I think I'll use lambdas instead and capture data more explicitly.